src/test/ui/const-generics/raw-ptr-const-param.rs

```diff
@@ -1,3 +1,6 @@
+// normalize-stderr-64bit "0x00000000" -> "0xPREFIX"
+// normalize-stderr-32bit "0x" -> "0xPREFIX"
```

Comment:
The test might be more readable if you wrote `[PREFIX]` or `{PREFIX}` instead of just `PREFIX`.